test/unit/services/notification_delivery_service_test.rb

```diff
@@ -37,4 +37,16 @@ def test_invalid_event
       NotificationDeliveryService.new(notification).call
     end
   end
+
+  def test_false_boolean_attribute_is_valid_event
+    account = FactoryGirl.create(:simple_provider, buyer: false)
+    FactoryGirl.create(:active_admin, account: account)
+    event = Accounts::AccountDeletedEvent.create(account)
+    notification = FactoryGirl.build_stubbed(:notification, system_name: :account_deleted)
+    notification.expects(:parent_event).returns(event)
+
+    assert_nothing_raised NotificationDeliveryService::InvalidEventError do
```

Comment:
This is a mock it does not do anything and is removed from future test unit
Just do not `assert_nothing_raised` it has no value